### DISCUSSION TITLE
types of steps

### DISCUSSION THREAD

**Discussion by spgerenday** [giorgk/ichnos#9] [General]
Hi,
What is the relationship between StepSize and StepSizeTime in the configuration file? I see in the help menu that StepSize is in units of length and StepSizeTime is in units of time. Should they both be specified in the configuration file? Presumably in the same amount of time, a particle would travel a different distance in different parts of the velocity field. Also, do I assume correctly that the units are the same ones used in the velocity file?
Thanks

**Comment by giorgk** [comment]
Please have a look at the following link. It should give you some directions
https://github.com/giorgk/ichnos/wiki/Step-control

The units are the same in all files
